pages/docs/tutorials/getting-started/slackbot-websocket-tutorial.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Create an AsyncAPI Document for a SlackBot with WebSockets
```

Comment:
```suggestion
title: Create an AsyncAPI Document for a Slackbot with WebSockets
```

oops, let's be consistent with spelling.. check the rest of your document to ensure you spell Slackbot consistently and fix accordingly.